source/include/ota_private.h

```diff
@@ -119,7 +119,7 @@
  * @brief A composite cryptographic signature structure able to hold our largest supported signature.
  */
 
-#define kOTA_MaxSignatureSize           256 /* Max bytes supported for a file signature (2048 bit RSA is 256 bytes). */
```

Comment:
I think we should also change the struct name to match the change:

https://github.com/aws/ota-for-aws-iot-embedded-sdk/blob/1f2362f005f39a1847d645660ecaff93890cfcd7/source/include/ota_private.h#L372

Reply:
Fixed in new commit.